libs/compute_api/src/spec.rs

```diff
@@ -289,14 +283,25 @@ impl ComputeMode {
 }
 
 /// Log level for audit logging
-/// Disabled, log, hipaa
-/// Default is Disabled
 #[derive(Clone, Debug, Default, Eq, PartialEq, Deserialize, Serialize)]
 pub enum ComputeAudit {
     #[default]
     Disabled,
+    // Deprecated, use Base instead
     Log,
+    // (log = 'ddl', log_parameter='off')
```

Comment:
Does `log = 'ddl'` mean `log_statement = `ddl`) ? Let's spell it out in full

Reply:
And does `log_parameter='off'`  mean `log_parameter_max_length=0` ? What exactly does `log_parameter='on'` translate to?

Reply:
these comments are about [pgaudit](https://github.com/pgaudit/pgaudit?tab=readme-ov-file#settings) GUCs,
I updated comments to mention it explicitly.

Reply:
Ah, gotcha!